### PR TITLE
Responsive pages

### PR DESCRIPTION
**Pull request by kelmag**
added responsiveness to styled components

### CODE REVIEW

File: src/components/Container/styles.js [hunk 1]
```diff
@@ -7,6 +7,17 @@ export const Weediv = styled.div`
   margin: 0;
   max-width: 850px;
   min-width: 200px;
+  @media (max-width:1100px) {
+    margin: auto;
+    margin-bottom:20px;
+    padding-bottom:20px;
+    @media (max-width:400px) {
+      width: 380px;
+      @media (max-width:400px) {
```
Comment (abumalick) [inline src/components/Container/styles.js]: you should not interlock your media your media queries here.
```
 @media (max-width:1100px) {
    margin: auto;
    margin-bottom:20px;
    padding-bottom:20px;
}
@media (max-width:400px) {
   width: 290px;
}
```

File: src/components/Content/styles.js [hunk 2]
```diff
@@ -2,6 +2,14 @@ import styled from "styled-components";
 
 export const Weecontent = styled.div`
   margin: 100px;
+  @media (max-width:1100px) {
+    margin: 100px 60px;
+    flex-direction: column;
+    @media (max-width:570px) {
```
Comment (abumalick) [inline src/components/Content/styles.js]: same here

File: src/components/Container/styles.js [hunk 3]
```diff
@@ -7,7 +7,15 @@ export const Weediv = styled.div`
   margin: 0;
   max-width: 850px;
   min-width: 200px;
+  @media (max-width:1100px) {
```
Comment (bigfanjs) [inline src/components/Container/styles.js]: it's really good to know one beautiful thing even if it's not that needed for such small app.

you see this `1100px`. this is a screen size. You would definitely need to reuse that across the app. Styled components has advanced feature called theming

It allows you to create an application-specific theme. Create a theme object like this:

```
module.exports = {
   colors: {
        red: "#FF6347" // your application specific red color
    },
    fonts: {
        tiny: "6px",
        big: "20px"
     },
     screens: {
           tiny: "450px",
           small: "680px"
      }
}
```

and import that and use it in the heme Provider:

```
import theme from 'data/theme'

<ThemeProvider theme={theme}>
    <App />
</ThemeProvider>
```

### Usage:
```
@media (max-width: ${({ theme }) => (theme.screens.tiny)}) {
   /* Css Code for Tiny Screen Sizes */
}
```

Ref: https://www.styled-components.com/docs/advanced#theming

File: src/components/Content/styles.js (hunk 2 above)
Comment (bigfanjs) [reply src/components/Content/styles.js]: Important  Note from @abumalick

File: src/components/Container/styles.js (hunk 3 above)
Comment (kelmag) [reply src/components/Container/styles.js]: thats so coool, I'll definitely use it, thank you

**Pull request merged**